### ISSUE TITLE
Ideas to add to freedom

### ISSUE THREAD

**Issue by Cyliocios** [Ciremun/freedom#95]
Im writing this to possibly add two ideas that I had in mind. My two ideas are: 
     - Mania,Taiko,CTB Replay Keys support
     - Unmod Hidden, Fade-In, HardRock etc.
These are my ideas for freedom, and if you have time, please consider these two ! I'm not entirely sure if the first idea is possible, but it'd be likely to get the timing atleast _close_ to it. Thank you

**Comment by GamebP** [comment]
Yeah.. but he only maintains the standart gamemode. He might in future make it for other gamemodes. But I doubt it.
